src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -254,13 +254,9 @@ public function get_grid_days( $date = null, $force = false ) {
 				$start = tribe_beginning_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 				$end   = tribe_end_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 
-				$results_in_day = array_filter( $day_results, static function( $event ) use ( $start, $end ) {
-					return (
-						( $start >= $event->start_date && $end <= $event->end_date )
-						|| ( $start >= $event->start_date && $start <= $event->end_date && $end >= $event->end_date )
-						|| ( $start < $event->start_date && $end >= $event->end_date )
-						|| ( $start < $event->start_date && $end <= $event->end_date && $end >= $event->start_date )
-					);
+				// Events overlap a day if Event start date <= Day End AND Event end date >= Day Start.
+				$results_in_day = array_filter( $day_results, static function ( $event ) use ( $start, $end ) {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: d8dcb8e4dbea47483df3eed4e5e70c3e</sub>